### ISSUE TITLE
OODBBean::__toString() implementation only works with C_ERR_IGNORE

### ISSUE THREAD

**Issue by Bernhard---H** [gabordemooij/redbean#512]
The implementation of the method `RedBeanPHP\OODBBean::__toString()` only works as intended if `\R::setErrorHandlingFUSE(\RedBeanPHP\OODBBean::C_ERR_IGNORE);` is set.

If a model class doesn't implement the `__toString()` method, the `RedBeanPHP\OODBBean::__call()` method will at least cause (possible confusing) spam in in the log files or fail unexpectedly.

And even worse, in case of `\RedBeanPHP\OODBBean::C_ERR_EXCEPTION` php will fail with 

> PHP Fatal error:  Method RedBeanPHP\OODBBean::__toString() must not throw an exception

and no hint on how to debug that.

---

I temporarily fixed it in the method `RedBeanPHP\OODBBean::__call()` by adding the `__toString` method 
to the whitelist:

``` php
if ( in_array( $method, array( '__toString', 'update', 'open', 'delete', 'after_delete', 'after_update', 'dispense' ), TRUE ) ) {
    return NULL;
}
```
But I have no idea if that destroys anything else.

**Comment by gabordemooij** [comment]
Sorry for the delay - I had some trouble at work and I started my own company.
I'll look into this.

**Comment by gabordemooij** [comment]
Fixed in c2520bbc3e6e68e78eff4af0acede9e93ab296b1
